### PR TITLE
Add python recording and regression slides.

### PR DESCRIPTION
**Pull request by angela24680403**
This PR adds the Intro to Python session, Solution pt 1 and Solution pt 2 recording links, as well as links to slides for Regression:
Intro to Python Page
Screenshot 2023-10-22 at 21 38 28
Regression Page
Screenshot 2023-10-22 at 21 38 08

### CODE REVIEW

File: our-initiatives/tutorials/intro-to-python.md [hunk 1]
```diff
@@ -14,6 +14,13 @@ Here are the **exercises** you can go through: 📘 [**Tutorial 1 Exercises**](h
 
 You can access our **slides** here: 💻 [**Tutorial 1 Slides**](https://www.canva.com/design/DAFmvE-ptx0/lyY0SiOcjgSxrb201KcC8w/edit)
 
+The **recording** from this session is available here: 🎤 [**Tutorial 1 Recording**](https://www.youtube.com/watch?v=adnK-hc2DmY)
+
+Here are two videos covering our exercise **solutions**:
+
+- Exercises 1 to 4: [Solution pt1](https://www.youtube.com/watch?v=qmHgg0IR680&t=116s)
```
Comment (TheRootOf3) [inline our-initiatives/tutorials/intro-to-python.md]: I don't think this was intential (if it was, please ignore) but this link makes this video start 116 seconds after the beginning. Suggestion below!
```suggestion
- Exercises 1 to 4: [Solution pt1](https://www.youtube.com/watch?v=qmHgg0IR680)
```

File: our-initiatives/tutorials/intro-to-python.md [hunk 2]
```diff
@@ -14,6 +14,13 @@ Here are the **exercises** you can go through: 📘 [**Tutorial 1 Exercises**](h
...
+- Exercises 5 and 6: [Solution pt2](https://www.youtube.com/watch?v=3Kmt0qmibrg&t=150s)
```
Comment (TheRootOf3) [inline our-initiatives/tutorials/intro-to-python.md]: Same thing as above. If intentional, please ignore!
```suggestion
- Exercises 5 and 6: [Solution pt2](https://www.youtube.com/watch?v=3Kmt0qmibrg)
```

**Review by TheRootOf3 — CHANGES_REQUESTED** [review]
Please have a look at my comments and either ignore or fix!

File: our-initiatives/tutorials/intro-to-python.md (hunk 1 above)
Comment (angela24680403) [reply our-initiatives/tutorials/intro-to-python.md]: Oh yes my bad thanks!

**Review by TheRootOf3 — APPROVED** [review]
LGTM 🎉

**Pull request merged**